internal/test/integration/red_test_ruby.go

```diff
@@ -167,3 +167,51 @@ func testHTTPTracesNestedNginx(t *testing.T) {
 		}, test.Interval(100*time.Millisecond))
 	}
 }
+
+// Assumes we've run the metrics tests
+func testHTTPTracesNestedNginxSQL(t *testing.T) {
+	for i := 1; i <= 4; i++ {
+		go ti.DoHTTPGet(t, "https://localhost:8443/users/"+strconv.Itoa(i), 200)
```

Comment:
do these need to be ran async?

Reply:
yeah, if we wait one by one it's always using the same thread, I needed to ensure these run in parallel to expose the problem of multiple queuing requests.